DDDEastAnglia.Tests/Helpers/Email/SendGrid/HtmlRendererFacts.cs

```diff
@@ -0,0 +1,40 @@
+﻿using DDDEastAnglia.Helpers.Email.SendGrid;
+using NUnit.Framework;
+
+namespace DDDEastAnglia.Tests.Helpers.Email.SendGrid
+{
+    [TestFixture]
+    public class HtmlRendererFacts
+    {
+        [TestFixture]
+        public class Render_Should
+        {
+            [Test]
+            public void Convert_Markdown_To_HTML()
+            {
+                var sut = CreateSut();
```

Comment:
I just find it *very* distracting. I read it as "do something, but I don't know what, then use the thing it returned that I don't know what it is and call a method `Render` on it". All of the vagueness means I find it harder to reason about the test.

In some other tests below, you've named the variable something related to what it is and that is less distracting.